Source/Core/Core/AchievementManager.cpp

```diff
@@ -386,8 +387,8 @@ bool AchievementManager::IsHardcoreModeActive() const
 }
 
 template <typename T>
-void AchievementManager::FilterApprovedIni(std::vector<T>& codes,
-                                           const std::string& game_ini_id) const
+void AchievementManager::FilterApprovedIni(std::vector<T>& codes, const std::string& game_ini_id,
```

Comment:
```suggestion
void AchievementManager::FilterApprovedIni(std::vector<T>& codes, const std::string& game_id,
```
Let's use the term that the rest of Dolphin uses, especially since we care about the distinction between the game ID and the name of the specific game INI.

(The same applies to other functions edited in this PR.)